python/lsst/ts/genericcamera/genericcameracsc.py

```diff
@@ -496,3 +750,13 @@ def _assert_live(self):
         """Raise an exception if live view is not active."""
         if not self.isLive:
             raise Exception()
+
+    def _assert_notautoexposure(self):
+        """Raise an exception if auto exposure is active."""
+        if self.isAutoExposure:
+            raise Exception()
+
+    def _assert_autoexposure(self):
+        """Raise an exception if auto exposure is not active."""
+        if not self.isAutoExposure:
```

Comment:
Same as above, do:

```
assert not self.isAutoExposure, "Auto exposure is not active."
```

Reply:
Following the live view design pattern and again I agree this is better. Done.

Reply:
Actually it should be `assert self.isAutoExposure, "Auto exposure is not active."` here.